src/sentry_core.c

```diff
@@ -692,3 +735,71 @@ sentry_set_level(sentry_level_t level)
         scope->level = level;
     }
 }
+
+sentry_value_t
+sentry_transaction_start(sentry_value_t tx_cxt)
+{
+    sentry_value_t tx = sentry_value_new_event();
+
+    // TODO: stuff transaction into the scope
```

Comment:
shouldn’t API users do that themselves rather?

Reply:
if we go with today's suggestion 3, then yes i think

Reply:
no: the transaction which is occasionally referred to as as the root span, and is a superset of a span is managed by the native SDK. transactions tend to contain things that track all of the child spans that spawn off them, and notably have `name`s which spans do not. in addition to that, i'm pretty sure that the native SDK is committing to only ever letting one transaction be active at once, which couples nicely with the way the scope is implemented and managed in this SDK.

spans, which are all children of a transaction will be managed by the end users. 

that was my understanding of the discussion, and is subject to my shoddy short term memory. feel free to correct or comment on this conclusion.